src/gui/owncloudgui.cpp

```diff
@@ -164,7 +151,14 @@ void ownCloudGui::slotOpenMainDialog()
 
 void ownCloudGui::slotTrayClicked(QSystemTrayIcon::ActivationReason reason)
 {
-    if (reason == QSystemTrayIcon::Trigger || reason == QSystemTrayIcon::Context) {
+#ifdef Q_OS_MAC
+    constexpr auto macos = true;
+#else
+    constexpr auto macos = false;
+#endif
+
+    // for mac os right and left click trigger the same behavior
+    if (reason == QSystemTrayIcon::Trigger || macos){
```

Comment:
*If* we still have a change in here (see my other comment), it really should be `|| (macos && reason == QSystemTrayIcon::Trigger)` and not just `|| macos`.

Reply:
Removed it.